src/app/entities/courses/directives/availability-status/availability-status.directive.ts

```diff
@@ -0,0 +1,36 @@
+import { Directive, HostBinding, Input } from '@angular/core';
+
+const FOURTEEN_DAYS_IN_MILLISECONDS = 1000 * 60 * 60 * 24 * 14;
+const COLORS = {
+  BLUE: '#009ECC',
+  GREEN: '#67A300',
+  TRANSPARENT: 'transparent',
+}
+
+const calculateBorderColor = (courseCreationDate: Date) => {
```

Comment:
It would be nice to move it somewhere, making it testable.
`getColorByDate` - here is nothing about Border. just dates and colors.

Reply:
@boale, I'll rename it. I think that it is tested implicitly in `availability-status.directive.spec.ts`. If this function doesn't work correctly the tests for `availability-status` directive will fail.